nemoguardrails/utils.py

```diff
@@ -302,3 +303,52 @@ def snake_to_camelcase(name: str) -> str:
         str: The converted CamelCase string.
     """
     return "".join(n.capitalize() for n in name.split("_"))
+
+
+def get_railsignore_path() -> Path:
+    """Helper to get railsignore path.
+
+    Returns:
+         Path: The.railsignore file path.
+    """
+    current_path = Path(__file__).resolve()
```

Comment:
it can end up in site-packages once the package is installed from PyPI

Reply:
Can you elaborate on what is meant here? I don't quite follow

Reply:
I think changing it to something like Path.cwd(). I will push some changes which will include this part.